src/docs/development/androidx-migration.md

```diff
@@ -62,7 +62,7 @@ To create a new project run:
 flutter create --androidx -t <project-type> <new-project-path>
 ```
 
-### How do I migrate my existing module project to AndroidX?
+### Add to App
```

Comment:
@xster @RedBrogdon Do we have a real marketing term for this?  We have "add-to-app", "existing app", "module".  "Module" is an overloaded term for nearly every platform and language.

Reply:
resolved offline